account_reversal/models/account_move.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2011 Alexis de Lattre <alexis.delattre@akretion.com>
+# Copyright 2011 Nicolas Bessi (Camptocamp)
+# Copyright 2012-2013 Guewen Baconnier (Camptocamp)
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    to_be_reversed = fields.Boolean(
```

Comment:
Hey @sbejaoui, just a reminder about the PR that will move this field into another new module.
see here https://github.com/OCA/account-financial-tools/pull/1771

Reply:
Oups @rousseldenis , sorry I thought this was @sbejaoui 's PR. Actually @sbidoul asked me to create a PR last Friday it seems that you already made the job 🙂 
@sbidoul should I close my PR? I that case we should close your issue https://github.com/OCA/account-financial-tools/issues/1761 in this PR message.

Reply:
Ow, my bad, I had not thought this would reappear as a migration.

I guess it makes sens to keep the name?

However, it seems this PR does not have the reversal_id field and navigation.

Reply:
On the other hand, the account_reversal name is a little bit misleading because the reversal feature is part of Odoo core.

Reply:
Hello @rousseldenis,
Could you just add the `reversal_id field` [field](https://github.com/OCA/account-financial-tools/pull/1771/commits/7f5f9f21355dda215d9f069add6124843228b8c2#diff-656b2aceb1c3e57e4f626cc6a3db29be015da78b9f1979dcb4920b831c44e0ccR18-R23)?
Thanks!